RPGMakerDecrypter.MVMZ/ProjectReconstructor.cs

```diff
@@ -10,6 +10,7 @@ public abstract class ProjectReconstructor
             "audio",
             "css",
             "data",
+            "dataex",
```

Comment:
Have you confirmed by yourself that there actually can be a directory with this name?